helper-cli/src/main/kotlin/commands/ListStoredScanResultsCommand.kt

```diff
@@ -46,6 +49,7 @@ internal class ListStoredScanResultsCommand : CliktCommand(
     ).convert { it.expandTilde() }
         .file(mustExist = true, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = true)
         .convert { it.absoluteFile.normalize() }
+        .default(ortConfigDirectory.resolve(ORT_CONFIG_FILENAME))
```

Comment:
Not sure how to handle this. OrtConfiguration.load() now expects a non-nullable File, as in almost all use cases a configuration file is present. This command is the only exception. So, I guess there are two options:
* Support again a nullable File in OrtConfiguration.load() and add a check there or
* Provide some default value for this command - this could even be something like File("."), which will be detected as a non-existing configuration file.

What do you prefer?

Reply:
@fviernau are you fine with keeping this code in `helper-cli` as-is?

Reply:
Yes. As 99% of the helper-cli does not use any ORT configuration I'd prefer addressing this consistently in a separate task if at all.

Reply:
Now I'm confused. I'd interpret your "Yes" as "Yes, you can keep the code as it is in this PR". But your following sentence sounds like the opposite. Can you please clarify?

Also, please note that we *cannot* keep the code here as it currently is in `master` as the configuration file is not nullable anymore. Do we need to so *something*, and this current solution seems to be the least invasive.

Reply:
Please interpret it as   "Yes, you can keep the code as it is in this PR"